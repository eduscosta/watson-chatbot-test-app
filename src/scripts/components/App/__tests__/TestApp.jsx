/* eslint-env jest */

import React from 'react';
import { shallow } from 'enzyme';

import { App } from '../App';

describe('<App />', () => {
  const tree = shallow(<App />);

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
