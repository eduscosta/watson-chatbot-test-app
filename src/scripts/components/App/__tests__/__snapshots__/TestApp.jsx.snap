// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<main
  className="main"
>
  <div
    className="controls"
  >
    <ul />
    <ul />
  </div>
  <div
    className="chatBox"
  >
    <ul />
    <form
      className="input"
      onSubmit={[Function]}
    >
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
      <button
        type="submit"
      >
        Send
      </button>
    </form>
  </div>
  <div
    className="error"
  >
    <button
      onClick={[Function]}
    >
      Ok.
    </button>
  </div>
</main>
`;
